Winget-AutoUpdate/User-Run.ps1

```diff
@@ -44,9 +44,9 @@ Get-NotifLocale
 #Set common variables
 $OnClickAction = "$WorkingDir\logs\updates.log"
 $Button1Text = $NotifLocale.local.outputs.output[11].message
-$Title = "Winget-AutoUpdate (WAU)"
-$Balise = "Winget-AutoUpdate (WAU)"
-$UserRun = $True
```

Comment:
@KnifMelti, I agree

Reply:
If not we must in **Start-NotifTask.ps1** introduce 
```
$caller = Get-ChildItem $MyInvocation.PSCommandPath | Select-Object -Expand Name
    if ($caller -eq "User-Run.ps1") {...}
```